.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+on:
+  schedule:
+    - cron: "0 0 * * *"
+jobs:
```

Comment:
Should we also run the acceptance tests on pull requests, I guess we'd have to spin up the next.js server within CI?